### DISCUSSION TITLE
Looking up other user information?

### DISCUSSION THREAD

**Discussion by adamshand** [pocketbase/pocketbase#5029] [Q&A] [answered]
My understanding is that non-admin users can only look up their own record from the `users` collection.

I'm building a simple photo sharing app which has a public profile page per user.  With the default `users` collection there is no way to lookup `name`, `username`, `bio` of another user.

I believe my options are:

1. Create a new `profiles` collection with a relation to users.  Store all public user information there.  Update the `photos` user relation to point at `profiles` instead of `users`.

2. Create a `public_users` view which only has the columns I want to be publicly accessible (`name`, `bio` etc).  It's nice to not have two places to update user information, but it looks like relation fields can't link to views so there's no way to expand `photos` records to include user information?

Am I understanding correctly or missing anything obvious?

Thank you for any pointers.
Adam.

**Comment by ganigeorgiev** [comment, marked as answer]
> My understanding is that non-admin users can only look up their own record from the users collection.

It depends on your `users` collection API rules. So a third option depending on your use case could be to change your List and View `users` API rules (_note that the user's email will be visible only for admins, the owner of the record and when `emailVisibility` for the specific user is explicitly set to `true`_).

**Reply by adamshand** [reply]
Ahhh, that might work.  Thanks!
